defaults/main.yml

```diff
@@ -25,3 +26,5 @@ ntp_restrict:
 ntp_cron_handler_enabled: false
 
 ntp_tinker_panic: false
+
+ntp_skip_install: false
```

Comment:
This makes little sense, you could use `tags` to skip install steps, e.g. tag everything that does not install as `config`.

Reply:
yes, thats another option. I however prefere to have control from yaml config instead of cmd parameters - or is there a way to set tags from yaml config?